zcash_client_sqlite/src/wallet.rs

```diff
@@ -1628,11 +1732,15 @@ mod tests {
     #[test]
     #[cfg(feature = "transparent-inputs")]
     fn put_received_transparent_utxo() {
+        use crate::testing::TestBuilder;
+
         let mut st = TestBuilder::new().build();
 
         // Add an account to the wallet
         let seed = Secret::new([0u8; 32].to_vec());
-        let (account_id, _usk) = st.wallet_mut().create_account(&seed).unwrap();
+        let birthday =
+            AccountBirthday::from_parts(st.sapling_activation_height(), Frontier::empty(), None);
```

Comment:
Is this intentionally not `AccountBirthday::from_sapling_activation`? It seems equivalent.

Reply:
Fixed in https://github.com/zcash/librustzcash/pull/935